api-authorization/server/authorization-middleware.js

```
@@ -0,0 +1,31 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+
+function authorizationMiddleware(req, res, next) {
+  /* your code here */
+  const authorization = req.headers['x-access-token'];
+  if (!authorization) {
+    res.status(401).json('authentication required');
```

Since this is middleware and you have error handling middleware installed, it's best to throw an exception here instead of returning status.
(`throw new ClientError(401, 'authentication required');`)